### DISCUSSION TITLE
How to use images as pagination instead of the bullets?

### DISCUSSION THREAD

**Discussion by palomamtnez** [nolimits4web/swiper#6191] [Q&A]
I'm trying to get an overview of how to approach the development for this block.
The images work as pagination, so they are clickable. The arrows are also clickable 
Any help is valuable! 

![Screen Shot 2022-11-08 at 10 32 27 PM](https://user-images.githubusercontent.com/13962572/200528290-b03a8abb-0038-47d0-a69f-1a45c0a36677.png)

**Comment by Arhell** [comment]
Hi
Try this
https://stackoverflow.com/questions/44115954/swiper-touch-events-enable-click-but-disable-drag
https://swiperjs.com/swiper-api#parameters allowTouchMove to false
